packages/vite/src/node/plugins/html.ts

```diff
@@ -590,72 +590,91 @@ function toPublicPath(filename: string, config: ResolvedConfig) {
   return isExternalUrl(filename) ? filename : config.base + filename
 }
 
-const headInjectRE = /<\/head>/
-const headPrependInjectRE = [/<head>/, /<!doctype html>/i]
+const headInjectRE = /([ \t]*)<\/head>/
+const headPrependInjectRE = [/([ \t]*)<head>/, /<!doctype html>/i]
```

Comment:
Shouldn't we use `\s`?

Reply:
I thought that `\s` would also match `\n` and `\r` that should be excluded as this group is trying to get the indentation of the tag